### ISSUE TITLE
allow for defining population, urban region, gtfs etc datasets in region configuration file directly

### ISSUE THREAD

**Issue by carlhiggs** [healthysustainablecities/global-indicators#298]
Currently a number of datasets are defined in the datasets.yml file, rather than directly in region.yml files.  The rationale for this was that these could be defined once and shared across multiple regions, potentially.  This was useful in our 25-city study, avoiding repetition in the city configuration files.  However, it does make configuration more fragmented and confusing, particularly when multiple analyses aren't being undertaken.  

In short, it would probably make things easier for new users to define all settings for a region in the one region file.  I think we should enable this as an option, as it will make it easier to demonstrate how to configure things.
